deno.json

```diff
@@ -1,3 +1,9 @@
 {
-  "importMap": "./import_map.json"
+  "name": "@denosaurs/rutt",
+  "version": "0.1.0",
+  "imports": {
+    "https://deno.land/x/rutt/mod.ts": "./mod.ts",
+    "jsr:@denosaurs/rutt": "./mod.ts"
+  },
```

Comment:
This existed before for the examples to run against locally as if you were importing them from the web. These will be interpreted by JSR as dependencies which they are not.

Reply:
Let's either remove them entirely or move them out to an import map and document why they exist for development. If you remove them, change the examples imports to point to `../mod.ts`.